product_lot_sequence/__manifest__.py

```diff
@@ -7,8 +8,8 @@
         Adds ability to define a lot sequence from the product""",
     "version": "14.0.1.0.2",
     "license": "AGPL-3",
-    "author": "ForgeFlow S.L., Odoo Community Association (OCA)",
+    "author": "Ooops, Cetmix, ForgeFlow S.L., Odoo Community Association (OCA)",
```

Comment:
Changing authors for this small change (and more adding 2 authors), it's too much. Anyway, the funding company should go to CREDITS.

Reply:
@pedrobaeza thanks, I agree, and I'm sorry we're wasting so much time with this stuff. 

When subcontracting development, we don't have control on these formal things, and the request for clearer guidelines goes in that direction > once the devs have a clear template to learn from, at their 3rd-4th PR iteration we expect everything to be correct. 

Together with the Functional documentation group we'll soon propose PRs to improve readme files to make it easier to understand DOs and DONTs.